GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -78,7 +78,7 @@ public partial class CommitInfo : GitModuleControl
         private List<string> _branches;
         private string _branchInfo;
         private string _gitDescribeInfo;
-        [CanBeNull] private IList<string> _sortedRefs;
+        [CanBeNull] private IDictionary<string, int> _sortedRefs;
```

Comment:
As this is no longer technically "sorted" maybe `_indexByRefName` is a better name for the field.